ios/browser/api/sync/driver/brave_sync_profile_service.h

```diff
@@ -24,7 +24,7 @@ typedef NS_OPTIONS(NSUInteger, BraveSyncUserSelectableTypes) {
   BraveSyncUserSelectableTypes_TABS = 1ULL << 10,
   BraveSyncUserSelectableTypes_SAVED_TAB_GROUPS = 1ULL << 11,
   BraveSyncUserSelectableTypes_PAYMENTS = 1ULL << 12,
-  BraveSyncUserSelectableTypes_SHARED_TAB_GROUP_DATA = 1ULL << 13,
+  // BraveSyncUserSelectableTypes_SHARED_TAB_GROUP_DATA removed
```

Comment:
Can be deleted?

Reply:
I am guessing @cdesouza-chromium left it as an explanation for the gap, but up to you, would you rather we deleted it?

Reply:
No problem removing it, this is just a wrapper type for Swift export anyways

Reply:
Removed in https://github.com/brave/brave-core/pull/28471/commits/afecdc09e129037f8debc774ba1517d52dde8d37